### PR TITLE
ci: implement automated release

### PR DESCRIPTION
**Pull request by barmac**
Related to https://github.com/bpmn-io/internal-docs/issues/639

### CODE REVIEW

**Comment by barmac** [comment]
@nikku One more thought: This will also create GitHub Releases. If it's not what we want to have, we can disable it. Personally, it doesn't bother me.

File: .github/workflows/POST_RELEASE.yml [hunk 1]
```diff
@@ -64,8 +64,3 @@ jobs:
         BPMN_IO_EMAIL: ${{ secrets.BPMN_IO_EMAIL }}
         BPMN_IO_USERNAME: ${{ secrets.BPMN_IO_USERNAME }}
       run: tasks/stages/update-website
-  update-translations:
```
Comment (nikku) [inline .github/workflows/POST_RELEASE.yml:67]: Is there a reason you moved this?

Comment (barmac) [reply .github/workflows/POST_RELEASE.yml]: I just moved it back because there was really no reason to do this.

**Comment by barmac** [comment]
> Why would we want a GitHub release? As long as it will be created consistently, without work from our end, I think we could have it.

They are created automatically by release-please. I did not do anything specifically to create those at https://github.com/bpmn-io/dmn-variable-resolver/releases

I can see some arguments to start using GH releases:

* If one follows the repository on GitHub, they will be notified about the releases. Unlike blog posts, we don't need to write it manually.
* Disabling the feature is additional code.

I'd say the only argument against this is that we have not used GH releases outside of Camunda Modeler, i.e. tradition.

**Comment by nikku** [comment]
Notifications + collecting :heart: should be enough in 2023 to start :see_no_evil:, let's do it.

**Review by nikku — APPROVED** [review]

**Comment by barmac** [comment]
This is clearly not working right now. I will look into how we can improve the situation.

**Pull request merged**